spec/lib/diaspora_federation/entities/comment_spec.rb

```diff
@@ -0,0 +1,27 @@
+module DiasporaFederation
+  describe Entities::Comment do
+    let(:data) {
```

Comment:
It is common for almost all merged entities specs. Do you think it is really necessary to change this? If so, it's logical to change it to FactoryGirl everywhere.